### PR TITLE
demos how scroll should work on details pages (NOT FOR PRODUCTION)

### PR DESCRIPTION
**Pull request by johngruen**
@meandavejustice hey, this PR demos the intended scrolling behavior on details page. Obviously this is not for implementation, but i wanted to convey how it should work.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ gulp.task('scripts', ['lint'], function scriptsTask() {
   return bundledStream;
 });
 
-gulp.task('styles', ['sass-lint'], function stylesTask() {
+gulp.task('styles', function stylesTask() {
```
Comment (johngruen) [inline gulpfile.js:118]: removed linting because it was bugging me :)

File: idea_town/frontend/templates/idea_town/frontend/index.html [hunk 2]
```diff
@@ -10,5 +10,27 @@
...
+        });
+
+        function checkScroll() {
+          if ($(window).scrollTop() > detailsHeaderOffset && !detailsWrapper.hasClass('stick')) {
+            detailsWrapper.addClass('stick');
+          } else if ($(window).scrollTop() < detailsHeaderOffset && detailsWrapper.hasClass('stick')) {
+            detailsWrapper.removeClass('stick');
+          }
+        }
+      }, 150)
```
Comment (johngruen) [inline idea_town/frontend/templates/idea_town/frontend/index.html:31]: Time set arbitrarily to give the ampersand app time to load

File: idea_town/frontend/templates/idea_town/frontend/index.html [hunk 3]
```diff
@@ -10,5 +10,27 @@
 
   <script src="{{ static('app/app.js') }}"></script>
 
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```
Comment (johngruen) [inline idea_town/frontend/templates/idea_town/frontend/index.html:13]: Obviously we won't use any of this

**Comment by meandavejustice** [comment]
Thanks! implemented in #304

**Pull request closed without merging**